resource/lookup_pipeline.yaml

```diff
@@ -0,0 +1,46 @@
+---
+jobs:
+- name: deploy
+  plan:
+
+  - get: release
+    trigger: true
+    params:
+      include_source_tarball: true
+
+  - get: common-tasks
+
+  - task: lookup-source
+    file: common-tasks/extract-release-tarball.yaml
+    output_mapping: {extracted: lookup-source}
```

Comment:
I'm not sure, but it looks like you're extracting the tarball and then do nothing with it. Shouldn't there be an input to the run-lookup-etl task, or is it implicit?

Reply:
So does the output get mounted in to the `run-lookup-etl` job docker file? Or do I have to explicitly add the volume to the docker file in the job

Reply:
I just want one directory from the tarball